README.md

```diff
@@ -1,3 +1,6 @@
+[![Open in Gitpod](https://gitpod.io/button/open-in-gitpod.svg)](https://gitpod.io/#https://github.com/lolcookie/elm-spa-lamdera)
```

Comment:
Should this reference this repo instead?

Reply:
yep, updated